stride/plotting/plot_traces.py

```diff
@@ -50,6 +50,113 @@ def plot_trace(*args, axis=None, colour='black', line_style='solid', title=None,
     return axis
 
 
+def plot_magnitude_spectrum(sampling_rate, *args, skip=1, time_range=None, norm=True,
+                            colour='black', line_style='solid', title=None, axis=None,
+                            per_trace=False, **kwargs):
+    """
+    Plot the magnitude spectrum of wavelet traces in dB scale.
+
+    Parameters
+    ----------
+    sampling_rate : float
+        Sampling rate of the wavelet signal in Hz.
+    args : arrays
+        Optional trace ID grid, frequency grid, and signal data.
+    skip : int, optional
+        Traces to skip, defaults to 1.
+    time_range : tuple, optional
+        Time range to plot, defaults to all.
+    norm : bool, optional
+        Normalize the whole gather, defaults to True.
+    colour : str, optional
+        Colour for the traces, defaults to black.
+    line_style : str, optional
+        Line style for the plot, defaults to solid.
+    title : str, optional
+        Plot title, defaults to None.
+    per_trace : bool, optional
```

Comment:
Looking at the code below, it seems `per_trace` means something else. The name is not very descriptive, so I suggest changing to a different one. See also comment above about matching signature of `plot_gather`

Reply:
I'll adopt the string descriptor from the bottom options